### ISSUE TITLE
runtime of 3.1 GA not available on linux?

### ISSUE THREAD

**Issue by KamranShahid** [dotnet/runtime#506]
Not able to find runtime for .net core 3.1
https://docs.microsoft.com/en-us/dotnet/core/install/linux-package-manager-ubuntu-1804
Tried both .net runtime and asp.net runtime but package seems not to be found
I am neither able to get any list of package when tried a command 
sudo apt-get update

**Comment by Hoffs** [comment]
Same on centos,
```
======================= N/S matched: aspnetcore-runtime ========================
aspnetcore-runtime-2.1.x86_64 : Microsoft ASP.NET Core 2.1.14 Shared Framework
aspnetcore-runtime-2.2.x86_64 : Microsoft ASP.NET Core 2.2.8 Shared Framework
aspnetcore-runtime-3.0.x86_64 : Microsoft ASP.NET Core 3.0.1 Shared Framework
```

**Comment by KamranShahid** [comment]
> Same on centos,
> 
> ```
> ======================= N/S matched: aspnetcore-runtime ========================
> aspnetcore-runtime-2.1.x86_64 : Microsoft ASP.NET Core 2.1.14 Shared Framework
> aspnetcore-runtime-2.2.x86_64 : Microsoft ASP.NET Core 2.2.8 Shared Framework
> aspnetcore-runtime-3.0.x86_64 : Microsoft ASP.NET Core 3.0.1 Shared Framework
> =========================== N/S matched: dotnet-sdk ============================
> dotnet-sdk-2.1.x86_64 : Microsoft .NET Core SDK 2.1.802 2.1.802
> dotnet-sdk-2.2.x86_64 : Microsoft .NET Core SDK 2.2.402 2.2.402
> dotnet-sdk-3.0.x86_64 : Microsoft .NET Core SDK 3.0.101 3.0.101
> ```

What command you have run there? It might help me a bit

**Comment by Hoffs** [comment]
> What command you have run there? It might help me a bit

This is for centos yum, so `yum search "dotnet-sdk"`,
for apt-get it should be `apt-cache search "dotnet-sdk"`

**Comment by davidsh** [comment]
@leecow

**Comment by leecow** [comment]
There are repo publishing errors showing up on 18.04. Investigating and will contact the repo owners.

**Comment by dagood** [comment]
This should be fixed now on Ubuntu 18.04, per https://github.com/dotnet/core/issues/3961. 🎉 

We have https://github.com/dotnet/core/issues/3960 tracking the RPM distros with more detail. CentOS should be available now, but others aren't quite there yet. Closing this issue as a semi-duplicate of https://github.com/dotnet/core/issues/3960.
